bentoml/frameworks/detectron.py

```diff
@@ -0,0 +1,83 @@
+import os
+from bentoml.exceptions import MissingDependencyException
+from bentoml.service.artifacts import BentoServiceArtifact
+from bentoml.service.env import BentoServiceEnv
+
+
+class DetectronModelArtifact(BentoServiceArtifact):
+    def __init__(self, name):
+        super(DetectronModelArtifact, self).__init__(name)
+        self._file_name = name
+        self._model = None
+        self._aug = None
+
+    def set_dependencies(self, env: BentoServiceEnv):
+        env.add_pip_packages(['torch', "detectron2"])
+
+    def _model_file_path(self, base_path):
+        return os.path.join(base_path, self.name)
+
+    def pack(self, detectron_model):  # pylint:disable=arguments-differ
```

Comment:
This is missing the `metadata` option.

You can add it similar to CoreML artifact:
https://github.com/bentoml/BentoML/blob/df114945f4046f112a4dc0b7afe5e8f80c2d58f8/bentoml/frameworks/coreml.py#L71